packages/features/embed/Embed.tsx

```diff
@@ -721,8 +739,9 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
           {eventTypeData?.eventType && embedType === "email" ? (
             <EmailEmbed
               eventType={eventTypeData?.eventType}
-              username={data?.user.username as string}
+              username={teamSlug ?? (data?.user.username as string)}
```

Comment:
Need to pass teamSlug and isTeamEvent to make useScheduledForEvent get event type details and slots